cmd/server/assets/apikeys/show.html

```diff
@@ -61,118 +61,122 @@ <h1>{{$authApp.Name}} API key</h1>
             Device (can verify codes)
           {{else if $authApp.IsAdminType}}
             Admin (can issue codes)
+          {{else if $authApp.IsStatsType}}
+            Stats (can view stats)
           {{else}}
             Unknown
           {{end}}
         </div>
       </div>
     </div>
 
-    <div class="card mb-3 shadow-sm">
-      <div class="card-header">
-        <span class="oi oi-bar-chart mr-2 ml-n1"></span>
-        Statistics
-      </div>
-      <div class="card-body">
-        <div id="apikey_stats_chart" class="container d-flex h-100 w-100" style="min-height:300px;">
-          <p class="justify-content-center align-self-center text-center font-italic w-100">Loading chart...</p>
+    {{if $authApp.IsAdminType}}
```

Comment:
This is all just indented ([see](https://github.com/google/exposure-notifications-verification-server/pull/1404/files?w=1))